controller/space.go

```diff
@@ -325,6 +330,9 @@ func ConvertSpace(request *goa.RequestData, p *space.Space, additional ...SpaceC
 		},
 		Links: &app.GenericLinks{
```

Comment:
done in https://github.com/almighty/almighty-core/pull/1033/commits/55fd30c0e78ab5e0df28e05c1334f4d1eb524478